### DISCUSSION TITLE
Release v1.10.4 / 5.65.4

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3187] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build adds a new scriptable troubleshooting wizard to help resolve sandboxing and UI issues, the wizard can be directly invoked from the SBIEMSG popup dialog. The settings dialog has been slightly restructured to add more space to update options, a bug was fixed causing the updater to run every day and not once per week, and a setting was added allowing to pick the update interval.
We have also added an addon manager which allows to install additional components, like a script debugger for the troubleshooting wizard, a file checker, the logapi dll, and a few other useful addons.
This build has also significantly reworked the low level injection mechanism, it should now work better and is much more flexible, although this being a large change requires some testing to ensure it works great on all scenarios.
For a full list of changes and fixes, please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1100--5650---2023-07-12).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.10.4'>Release v1.10.4 / 5.65.4</a>.</em>

**Comment by BilleBarrett** [comment]
Looks OK but Microsoft Edge loads and then unloads, Edge did an update I think last night
Microsoft Edge
Version 115.0.1901.203 (Official build) (64-bit)
It runs OK unsandboxed :-(
can I look somewhere to find out what is wrong
